### PR TITLE
Collections Show page should use render_collection_links.  Fixes #1128

### LINKED ISSUE #1128: Collections Show page should use render_collection_links
Change: 
https://github.com/projecthydra/sufia/blob/pcdm/app/views/collections/_show_document_list_row.html.erb#L22

Add some tests to make sure the view renders a collection.  Seems to be missing or invalid since the page shows nothing now.

### PR DESCRIPTION
**Pull request by scherztc**
I could adjust the test to include generic_works if we remove file_format from app/views/collections/_show_document_list_row.html.erb.

### CODE REVIEW

File: spec/views/collections/_show_document_list_row.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      allow(view).to receive(:blacklight_config).and_return(Blacklight::Configuration.new)
+      allow(view).to receive(:current_user).and_return(nil)
+      allow(file).to receive(:title_or_label).and_return("One Hundred Years of Solitude")
+      allow(file).to receive(:edit_people).and_return([])
+
+    end
+
+    it "should render collection_link for file" do
+      render(partial: 'collections/show_document_list_row.html.erb', locals: {document: file})
+      expect(rendered).to have_content 'One Hundred Years of Solitude'
```
Comment (cam156) [inline spec/views/collections/_show_document_list_row.html.erb_spec.rb]: @scherztc Not seeing a check for the collection here. What am I missing?

**Comment by cam156** [comment]
Sound like a plan.  File format shoul not be a part of a work.

File: spec/views/collections/_show_document_list_row.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+  context 'when not logged in' do
+    before do
+      allow(view).to receive(:blacklight_config).and_return(Blacklight::Configuration.new)
+      allow(view).to receive(:current_user).and_return(nil)
+      allow(work).to receive(:title_or_label).and_return("One Hundred Years of Solitude")
+      allow(work).to receive(:edit_people).and_return([])
+    end
+
+    it "should render collections links" do
```
Comment (scherztc) [inline spec/views/collections/_show_document_list_row.html.erb_spec.rb]: @cam156:  I still can't this simple view test to work.  Any ideas?

**Comment by cam156** [comment]
:+1: waiting on travis

**Pull request merged**